optaweb-employee-rostering-frontend/pom.xml

```diff
@@ -50,6 +50,16 @@
               <goal>install-node-and-npm</goal>
             </goals>
           </execution>
+          <execution>
+            <id>npm install lock-treatment-tool and run-node</id>
+            <phase>initialize</phase>
+            <goals>
+              <goal>npm</goal>
+            </goals>
+            <configuration>
+              <arguments>install @kie/lock-treatment-tool</arguments>
+            </configuration>
+          </execution>
```

Comment:
IDK... Is this neccessary? @Ginxo can't we move `locktt` completely to the `productized` profile as you did in https://github.com/kiegroup/optaweb-employee-rostering/pull/734?

Reply:
well locktt is not needed by community (unless this is built with no access to npmjs registry) I think `productized` profile is the proper place to add locktt.

Reply:
and this is additionally defined at #L66 by `<id>lock-treatment-tool execution</id>` , why is it needed again? (without `--`)